docs/CHANGELOG.asciidoc

```diff
@@ -236,33 +236,42 @@ Platform::
 * Reporting/diagnostics {kibana-pull}74314[#74314]
 * Removes the light gray border around the image in PDF reports {kibana-pull}78036[#78036]
 * Increases capture.timeouts.openUrl to 1 minute {kibana-pull}75207[#75207]
 Security::
 * Hides management sections based on cluster/index privileges {kibana-pull}67791[#67791]
-* xpack.encryptedSavedObjects.encryptionKey can now be rotated without losing access to existing encrypted Saved Objects (alerts, actions etc.). 
-Old key(s) can be moved to xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys to be used only to decrypt existing objects while new or updated objects will be encrypted using new primary encryption key. 
+* xpack.encryptedSavedObjects.encryptionKey can now be rotated without losing access to existing encrypted Saved Objects (alerts, actions etc.).
+Old key(s) can be moved to xpack.encryptedSavedObjects.keyRotation.decryptionOnlyKeys to be used only to decrypt existing objects while new or updated objects will be encrypted using new primary encryption key.
 Administrators can also use dedicated API endpoint /api/encrypted_saved_objects/_rotate_key to trigger re-encryption of all existing objects with a new primary key so that old keys can be safely disposed {kibana-pull}72420[#72420]
 * Groups features for space management {kibana-pull}74151[#74151]
 * Allows passwords to be visible on security screens {kibana-pull}77394[#77394]
 * Groups features for role management {kibana-pull}78152[#78152]
 * Warns users when security is not configured {kibana-pull}78545[#78545]
 * Sharing saved-objects phase 1.5 {kibana-pull}75444[#75444]
 * [Detections] Handle conflicts on alert status update {kibana-pull}75492[#75492]
-* Improves the experience when Kibana returns a 403 HTTP status code or the user tries to access a page/app they do not have access to. In those instances, a new user-friendly error page is shown. 
-The user will get the option to go back to the page from where they came, or log in as a different user. Previously the user would just see a simple JSON document containing a short error message 
+* Improves the experience when Kibana returns a 403 HTTP status code or the user tries to access a page/app they do not have access to. In those instances, a new user-friendly error page is shown.
+The user will get the option to go back to the page from where they came, or log in as a different user. Previously the user would just see a simple JSON document containing a short error message
 without the ability to do anything {kibana-pull}75538[#75538]
 * Adds EQL search strategy {kibana-pull}78645[#78645]
 * Fetches related events from the server {kibana-pull}78780[#78780]
 * [Resolver] Requests data from new event api {kibana-pull}78782[#78782]
 * Updates copy styling {kibana-pull}79313[#79313]
 * Excludes cloud alias index from our query {kibana-pull}81551[#81551]
+* Implements server-side sessions. {kib} now stores user session information in a dedicated {es} index.
```

Comment:
This first sentence belongs in the Release Notes, but the remainder of the content belongs in the product guide. This is too long for a Release Notes entry.

Reply:
@KOTungseth I see other entries in the RN that are longer.  Can we keep some of the description here, but reduce it?

Reply:
I see what you mean. My only concern is that this information is buried, so users might not think to look for it here. I say merge it, but going forward, I'm going to do better with editing these long RN entries.